src/OrbitBase/include/OrbitBase/File.h

```diff
@@ -27,18 +27,18 @@ namespace orbit_base {
 
 constexpr int kInvalidFd = -1;
 
-class unique_fd {
+class UniqueFd {
```

Comment:
I somehow remember we discussed this in the past, the reason for the snake_case was to be similar to unique_ptr. I don't think I agreed with that and I prefer the new way.

Reply:
Same. I think it's unique_fd because it's unique_fd in android-base.